src/main/java/gov/usgs/wma/statistics/model/JsonDataBuilder.java

```diff
@@ -67,16 +68,21 @@
 	 * This is a collection of all the normalized values.
 	 * Is used for the overall percentile value.
 	 */
-	List<Value> intermediateValuesList = new LinkedList<>();
+	List<Value> intermediateValuesList   = new LinkedList<>();
```

Comment:
Looks like there's a bit of extra whitespace before the `=` here.

Reply:
Ah, I see what you're doing, this is intentional spacing. Ignore!

Reply:
yes, I suppose I could have not aligned them.